src/cloud/live/sens-data-over.md

```diff
@@ -170,8 +183,10 @@ This process **overwrites** the store configuration; only do the following if th
 [app-yaml]: {{ site.baseurl }}/cloud/project/magento-app.html
 [commerce-dump]: {{ site.baseurl }}/guides/v2.3/reference/cli/magento-commerce.html#appconfigdump
 [env-yaml]: {{ site.baseurl }}/cloud/project/magento-env-yaml.html
+[environment configuration variables]: {{ site.baseurl }}/cloud/project/projects.html#environment-configuration-variables
 [export]: {{ site.baseurl }}/config-guide/cli/config-cli-subcommands-config-mgmt-export.html
 [Optimize deployment]: {{ site.baseurl }}/cloud/deploy/optimize-cloud-deployment.html
 [Pipeline deployment]: {{ site.baseurl }}/guides/v2.3/config-guide/deployment/pipeline/technical-details.html
 [quick]: {{ site.baseurl }}/cloud/env/variables-build.html#scd_strategy
-[scd]: {{ site.baseurl }}/cloud/deploy/static-content-deployment.html
\ No newline at end of file
+[scd]: {{ site.baseurl }}/cloud/deploy/static-content-deployment.html
+[Sensitive or system-specific settings]: {{ site.baseurl }}/guides/v2.4/config-guide/cli/config-cli-subcommands-config-mgmt-set.html
```

Comment:
This seems to be a better link: https://devdocs.magento.com/guides/v2.4/config-guide/prod/config-reference-sens.html

```suggestion
[Sensitive or system-specific settings]: {{ site.baseurl }}/guides/v2.4/config-guide/prod/config-reference-sens.html
```

Reply:
@meker12 added this - but I concur it's the one you have said - adding @meker12 so they know we have agreed to change it.